Modify rules.csv with updates from PHL

govvin opened the pull request:
Modify rules.csv with initial (and partial) updates from the Philippines

govvin committed:
Modify rules.csv with updates from PHL

Modify rules.csv with initial (and partial) updates from the Philippines

PanierAvide commented on rules.csv:
```diff
@@ -801,3 +801,198 @@ US,supermarket,Fred Meyer,Q5495932,open_adapted,07:00-21:00,,Dedicated to senior
+PH,amenity=fuel,Caltex,Q277470,partial,,,,https://caltex.com/ph/covid-19-updates.html,
+PH,shop=pawnbroker,Cebuana Lhuillier,Q17064661,partial,,,"Opening hours vary based on location. Masbate,Eastern Visayas and Siquijor branches have extended hours.",,https://www.cebuanalhuillier.com/cebuana-lhuillier-advisory-on-operating-and-closed-branches-starting-march-2020/
+PH,shop=convenience,"Central Convenience Store
+",,partial,,,,,
```
Can you remove this line return ?

PanierAvide commented:
Thanks for this pull request, just a few remarks :
- Can you put all added lines in order to keep the whole file sorted alphabetically by country code, category, brand name ?
- Categories should only be a single string value, following OSM value conventions (so shop=bakery -> bakery), can you change accordingly ?

govvin committed:
Modify rules.csv with updates from Philippines

Modify rules.csv with initial (and partial) updates from the Philippines, including changes made after the feedback of the draft.

govvin commented:
Thank you for the feedback  @PanierAvide . I will update the request and submit a new PR.
